src/fsharp/IlxGen.fs

```diff
@@ -5150,7 +5196,7 @@ and GenLetRecFixup cenv cgbuf eenv (ilxCloSpec: IlxClosureSpec, e, ilField: ILFi
 
 /// Generate letrec bindings
 and GenLetRecBindings cenv (cgbuf: CodeGenBuffer) eenv (allBinds: Bindings, m) =
-    let eenv = SetIsInLoop true eenv
```

Comment:
Can you undo the renaming here to minimize the diff please?

Reply:
Sure, that's fair. I'll do the same to the other changes.